packages/wordpress-playground-block/src/components/playground-preview/index.tsx

```diff
@@ -390,8 +401,18 @@ export default function PlaygroundPreview({
 										}`}
 										aria-label={
 											isErrorLogFile(file)
-												? `Read-only file: ${file.name}`
-												: `File: ${file.name}`
+												? // translators: %s is a file name
+												  sprintf(
```

Comment:
I don't like how this text ended up being auto-formatted within the ternary expression. It might be better to assign to vars and just use them here for a more readable expression.

Reply:
Actually, these look fine in the editor but were just awkward during code review on GH. Will leave them as inline for now.